amethyst_core/src/util.rs

```diff
@@ -0,0 +1,53 @@
+use hibitset::BitSetLike;
+use specs::{storage::UnprotectedStorage, Component};
+
+pub trait Cache<T> {
```

Comment:
Shouldn't that be called `EntityCache`? If it was a generic cache, you would probably want a generic for the key.

Reply:
It's just utilty that's not exposed, plus it doesn't cache entities (but arbitrary data).

Reply:
**pub** trait Cache?

Reply:
~~pub~~ mod util; :wink: